### PR TITLE
Uptake latest go

### LINKED ISSUE #291: Uptake go1.20+
This also involves migrating from `crypto/elliptic` to `crypto/ecdh`, which is safer and more efficient.

### PR DESCRIPTION
**Pull request by decentralgabe**
Fixes #291

### CODE REVIEW

File: crypto/bbs.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import (
+	"crypto/rand"
+	"crypto/sha256"
+
+	bbs "github.com/hyperledger/aries-framework-go/pkg/crypto/primitive/bbs12381g2pub"
+)
+
+// GenerateBBSKeyPair https://w3c-ccg.github.io/ldp-bbs2020
+func GenerateBBSKeyPair() (*bbs.PublicKey, *bbs.PrivateKey, error) {
```
Comment (andresuribe87) [inline crypto/bbs.go:11]: Did you mean to have this be part of this PR?

**Review by andresuribe87 — COMMENTED** [review]
Are you intending to replace the `crypto/elliptic` occurrences in a different PR?

File: crypto/bbs.go (hunk 1 above)
Comment (decentralgabe) [reply crypto/bbs.go:11]: nope

**Comment by decentralgabe** [comment]
@andresuribe87 the update is for `The new [crypto/ecdh](https://go.dev/pkg/crypto/ecdh) package provides explicit support for Elliptic Curve Diffie-Hellman key exchanges over NIST curves and Curve25519.` the code we have doesn't use ECDH

**Comment by andresuribe87** [comment]
@decentralgabe I was thinking of the crypto/keys.go file as below https://github.com/TBD54566975/ssi-sdk/blob/d48966db0f15c5cb32efcc4233d2eb5368adb1ee/crypto/keys.go#L93 ?

**Comment by decentralgabe** [comment]
> https://go.dev/pkg/crypto/ecdh

Since we're not using ECDH (just ECDSA for now) there aren't any code changes to make. When/if we uptake ECDH we'll make use of this package.

**Review by andresuribe87 — APPROVED** [review]

**Pull request merged**